src/witan/httpapi/api.clj

```diff
@@ -298,7 +298,7 @@
               (success ACCEPTED r headers)
               (fail s))))
 
-        (PUT "/files/:file-id" req
+        (POST "/files/:file-id" req
```

Comment:
why the switch from PUT to POST?

Reply:
more in line with other routes (i.e. PUT is only used to create new metadata, all other ops (updating metadata, adding people to sharing matrix are done with POST)